Vizzuality/half-earth-v3
# Epic/update data layers

## Update data layers
### Description
This PR updates the data behind the map.
Some layers overlaid on the map have been updated:
  - __Human encroachment__ data has been updated so the new raster tiled service matches the data behind the calculation of the priority places for conservation.
  - __Terrestrial and marine global biodiversity data__ has been updated and calculated with the `~55km` grid instead of the `~110km` grid.

An important update concerning the __Grid__ driving the landscape mode calculations has been made. We are using now the updated `~55km` terrestrial grid  to calculate the species, protected areas and human pressures data on the landscape sidebar. We are still using the old grid in order to get marine data, but a dedicated marine grid would be available in the future. This update derived in some changes to the widgets of the landscape sidebar.

The `metadata` behind the layers has been also updated and we are now consuming it from a layer stored on the ArcGIS online organization, derived from the metadata stored on the layers themselves. This is the first step on the aim of moving away from contentful.

### Testing instructions
Check a complete user flow on the data globe activating and deactivating the data layers and testing that the landscape view mode is working properly.
Display layers metadata modal by clicking on the `info` button displayed on layers selectors after toggling on a given layer. Also check the metadata modal by activating it through the legend items.
### Feature relevant tickets
Epic stories are available on the [Data update epic](https://www.pivotaltracker.com/n/projects/2321903/epics/4528593) pivotal ticket
